lib/restaurant.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+
+class Restaurant
+	
+	def send_confirmation_message
+		account_sid = 'AC57bd6a35fc71e021da4ca730b6d1294e' 
```

Comment:
You don't want to push your twilio api tokens to github. Store them locally as environmental variables.